app/src/main/res/values/strings.xml

```diff
@@ -24,6 +24,8 @@
     <string name="contacts_name">Contacts</string>
     <string name="contacts_call_on_click">Click to call contacts</string>
     <string name="search_results_options">Search results</string>
+    <string name="fuzzy_search_name">Enable fuzzy search</string>
```

Comment:
I personally would change "Fuzzy" in text to something else, as this is misleading.
Current search algorithm is fuzzy and the other one too.
Difference is that one is just not checking if words are similar.

Reply:
"Typo tolerant search"?

Reply:
or something containing "similar"